views/frontend/home_cv.php

```diff
@@ -198,14 +198,22 @@
     </div>
   </section>
 
-<?php
-    // Appel de la fonction permettant de traiter le formualire de la page d'accueil "home_cv".        
-    $model_form->form_page_home_cv()
-?>
-
 <!--FOOTER DESCRIPTION -->
   <section id="contact">
 
+                     <?php if(!empty($errors)){
+                    ?>
+                        <div class="card red">
+                            <div class="card-content white-text">
+                                <?php
+                                foreach($errors as $error){
+                                    echo $error."</br>";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$error'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)